dev-tools/ci-integration.sh

```diff
@@ -1,6 +1,8 @@
-#!/usr/bin/env bash
+#!/bin/sh
 set -e
 
+IFS='
```

Comment:
just wondering... would
`printf -v IFS "\n"`
work ?

Reply:
`printf -v` is a bashism.

You mean `$'...'` as in `IFS=$'\n'`? No, `$'...'` is also a bashism.